cmd/allorad/cmd/commands.go

```diff
@@ -29,9 +28,6 @@ import (
 )
 
 func initRootCmd(rootCmd *cobra.Command, txConfig client.TxConfig, basicManager module.BasicManager) {
-	cfg := sdk.GetConfig()
```

Comment:
I don't have any idea what these lines do I'm pretty sure we copy pasted them from a cosmos tutorial somewhere

do you have a reason you're deleting them?